.github/workflows/turborepo-release.yml

```diff
@@ -193,15 +193,6 @@ jobs:
         with:
           github-token: "${{ secrets.GITHUB_TOKEN }}"
 
-      - name: Install GoReleaser
```

Comment:
We still need this, we haven't ripped out `goreleaser` yet. `make publish-turbo` invokes it.